.jenkins/mac-wheel.bat

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash -xe
```

Comment:
This should not have a `.bat` extension, should be `.sh`.

Also does this need `/bin/bash` or can `/bin/sh` work?